doc/01-running-spacegraphcats.md

```diff
@@ -160,9 +163,9 @@ twofoo_k31_r1_search_oh0/2.fa.gz.cdbg_ids.contigs.fa.gz
 twofoo_k31_r1_search_oh0/47.fa.gz.cdbg_ids.contigs.fa.gz
 twofoo_k31_r1_search_oh0/63.fa.gz.cdbg_ids.contigs.fa.gz
 
-twofoo_k31_r1_search_oh0/2.fa.gz.cdbg_ids.reads.fa.gz
-twofoo_k31_r1_search_oh0/47.fa.gz.cdbg_ids.reads.fa.gz
-twofoo_k31_r1_search_oh0/63.fa.gz.cdbg_ids.reads.fa.gz
+twofoo_k31_r1_search_oh0/2.fa.gz.cdbg_ids.reads.gz
```

Comment:
maybe indicate that these are either FASTA or FASTQ, depending on your input files?